### PR TITLE
Expose shrink on load in resize

### PR DESCRIPTION
**Pull request by rnanwani**

### CODE REVIEW

**Comment by rnanwani** [comment]
Not very sure how to write a test case for this. The two images are visibly different, but after resizing in assertSimilar, no difference is left.

File: test/fixtures/index.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ module.exports = {
   inputJpgLarge: getPath('giant-image.jpg'),
   inputJpg320x240: getPath('320x240.jpg'), // http://www.andrewault.net/2010/01/26/create-a-test-pattern-video-with-perl/
   inputJpgOverlayLayer2: getPath('alpha-layer-2-ink.jpg'),
+  inputJpgShrinkOnLoad: getPath('shrink-on-load-test.jpg'),
```
Comment (lovell) [inline test/fixtures/index.js]: What is the source of this test image? Are we granted the right to distribute it, e.g. via a suitable Creative Commons licence?

**Comment by lovell** [comment]
The change being proposed in #954 may supersede this.

**Comment by lovell** [comment]
This has been superseded by #977, which will be in v0.19.0. It exposes a `fastShrinkOnLoad` option to the `resize()` function, defaulting to `true`.

**Pull request closed without merging**